docs/statistics.md

```diff
@@ -314,266 +472,230 @@ either already acknowledged or dropped by TSBPD as too late to be delivered.
 
 Retransmitted packets can also be considered late.
 
-## pktSndDrop
+### pktSndDrop
 
-Same as `pktSndDropTotal`, but for a specified interval.
+Same as `pktSndDropTotal`, but for a specified interval. Available for sender.
 
-## pktRcvDrop
+### pktRcvDrop
 
-Same as `pktRcvDropTotal`, but for a specified interval.
+Same as `pktRcvDropTotal`, but for a specified interval. Available for receiver.
 
-## pktRcvUndecrypt
+### pktRcvUndecrypt
 
-Same as `pktRcvUndecryptTotal`, but for a specified interval.
+Same as `pktRcvUndecryptTotal`, but for a specified interval. Available for receiver.
 
-## byteSent
+### byteSent
 
-Same as `byteSentTotal`, but for a specified interval.
+Same as `byteSentTotal`, but for a specified interval. Available for sender.
 
-## byteRecv
+### byteRecv
 
-Same as `byteRecvTotal`, but for a specified interval.
+Same as `byteRecvTotal`, but for a specified interval. Available for receiver.
 
-## byteRcvLoss
+### byteRcvLoss
 
-Same as `byteRcvLossTotal`, but for a specified interval.
+Same as `byteRcvLossTotal`, but for a specified interval. Available for receiver.
 
-## byteRetrans
+### byteRetrans
 
-Same as `byteRetransTotal`, but for a specified interval.
+Same as `byteRetransTotal`, but for a specified interval. Available for sender.
 
-## byteSndDrop
+### byteSndDrop
 
-Same as `byteSndDropTotal`, but for a specified interval.
+Same as `byteSndDropTotal`, but for a specified interval. Available for sender.
 
-## byteRcvDrop
+### byteRcvDrop
 
-Same as `byteRcvDropTotal`, but for a specified interval.
+Same as `byteRcvDropTotal`, but for a specified interval. Available for receiver.
 
-## byteRcvUndecrypt
+### byteRcvUndecrypt
 
-Same as `byteRcvUndecryptTotal`, but for a specified interval.
+Same as `byteRcvUndecryptTotal`, but for a specified interval. Available for receiver.
 
-# Instant measurements
 
-The measurements effective at the time retrieved.
+## Instantaneous Statistics
 
-## usPktSndPeriod
+### usPktSndPeriod
 
-Current minimum time interval between which consecutive packets are sent, in 
-microseconds. Sender only.
+TODO: 
 
-Note that several sockets sharing one outgoing port use the same sending queue.
-They may have different pacing of the outgoing packets, but all the packets will
-be placed in the same sending queue, which may affect the send timing.
+- How is this calculated? The minimum time during which period?
```

Comment:
This isn't calculated, it's the really used control value that represents the time interval between two consecutive sent packets. That is, when a sending thread is about to send a packet and the time since last sending is less than this, it sleeps up to that time.

Note that packets used for bandwidth probing and control packets are sent without respecting this interval, however (if I'm not missed) for bandwidth probing packets this haste time is recorded as spared time (this means longer sleeping for the next packet).